kind/hack/bindata.bzl

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Genrule wrapper around the go-bindata utility.
+# forked from https://github.com/kubernetes/kubernetes/blob/master/build/bindata.bzl
+# IMPORTANT: Any changes to this rule may also require changes to hack/generate.sh.
```

Comment:
Is there any trivial way to ensure that changes are mirrored in both places? This seems like it could trip people up in future/go unnoticed!

Reply:
I've been thinking about this, I think we can do something like:
- generate the file in both places
- do an intelligent diff (maybe import it and dump all the assets and compare them?)

Reply:
A naive diff of both files makes them not the same because they use different path computations / prefixes to get to the same thing (bazel is using full paths, not relative). Maybe we can fix that instead.

Reply:
So I now generate identically in bazel and I have a script verifying this hooked into the codegen verify job.